modules/standard/Math.chpl

```diff
@@ -543,13 +547,14 @@ module Math {
     return chpl_expm1(x);
   }
 
-  /* Multiply by an integer power of 2.
-     Returns x * 2**n.
-     */
+  /* Returns the value of the argument `x` multiplied by the number 2 raised to
+    the argument `n` power. */
```

Comment:
I think I'd still maintain the equation in this documentation, but the rewording seems reasonable otherwise

Reply:
Ok to restore the equation